.github/workflows/stale-pull-requests.yml

```diff
@@ -0,0 +1,39 @@
+name: Check stale pull requests
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 12 * * *'
+
+permissions:
+  issues: read
+  pull-requests: write
+
+env:
+  BEFORE_STALE: 14
+  BEFORE_CLOSE: 7
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v5
+        with:
+          debug-only: true
+          # disable issues
+          days-before-issue-stale: -1
+          days-before-issue-close: -1
+          stale-pr-message: |
+            This pull request is stale because it has been open for ${{ env.BEFORE_STALE }} days with no activity.
+            It will be closed in ${{ env.BEFORE_CLOSE }} days unless the `stale` label is removed.
+          close-pr-message: |
+            This pull request was closed due to a lack of activity for ${{ env.BEFORE_CLOSE }} days after it was stale.
+          stale-pr-label: stale
+          close-pr-label: closed-due-inactivity
+          days-before-pr-stale: ${{ env.BEFORE_STALE }}
+          days-before-pr-close: ${{ env.BEFORE_CLOSE }}
+          exempt-pr-labels: 'external contribution'
+          exempt-draft-pr: true
+          exempt-all-milestones: true
+          remove-stale-when-updated: true
+          operations-per-run: 60
```

Comment:
I'd increase this to 100 so that we can process all our current PRs in one go.

Reply:
Yes. But for some reason, this action is parsing issues even though we explicitly disable it. I'm thinking of using another one that I did in my tests that call the graphql api directly for the PRs instead of use this `actions/stale`, but it is still a bit incomplete (it does not do everything that the `actions/stale` does yet).

Reply:
Right. It probably fetches from the API first and filters only then. Might be worth a feature request or something.

I'd keep it simple for now and stick to this action but yeah, in the longer term we might actually want to switch to the other one (or build a custom one) because I think that just marking as stale is not what we want in some cases. E.g. for unreviewed PRs we might want to instead assign someone randomly or ping the channel or something.